codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/IntermediateModel.java

```diff
@@ -169,6 +180,10 @@ public boolean getHasWaiters() {
         return waiters.size() > 0;
     }
 
+    public boolean getHasPaginators() {
```

Comment:
This weird naming seems like something we would do for FTL. Do we need it for the poet approach?

Reply:
waiters has the same naming. I thought it was used for serialization/deserialization. We might not need it. I can change it to hasPaginators()